packages/client/src/generation/generateClient.ts

```diff
@@ -301,18 +302,21 @@ export async function generateClient({
     await writeFile(packageJsonTargetPath, pkgJson)
   }
 
-  if (!testMode && process.env.INIT_CWD) {
-    const backupPath = path.join(process.env.INIT_CWD, 'node_modules/.prisma/client')
-    if (finalOutputDir !== backupPath && !generator?.isCustomOutput) {
-      await copy({
-        from: finalOutputDir,
-        to: backupPath,
-        recursive: true,
-        parallelJobs: process.platform === 'win32' ? 1 : 20,
-        overwrite: true,
-      })
-    }
-  }
+  // if (!testMode && process.env.INIT_CWD) {
+  //   const backupPath = path.join(process.env.INIT_CWD, 'node_modules/.prisma/client')
+  //   console.log('backupPath', backupPath)
+  //   console.log('finalOutputDir', finalOutputDir)
+
+  //   // if (finalOutputDir !== backupPath && !generator?.isCustomOutput) {
+  //   //   await copy({
+  //   //     from: finalOutputDir,
+  //   //     to: backupPath,
+  //   //     recursive: true,
+  //   //     parallelJobs: process.platform === 'win32' ? 1 : 20,
+  //   //     overwrite: true,
+  //   //   })
+  //   // }
+  // }
```

Comment:
Should we just remove dead code or leave it here commented out until we investigate?

Reply:
Thanks, I forgot to remove it.

Reply:
```suggestion
```